README.md

```diff
@@ -31,26 +32,65 @@ Read back and parse the file using `cabinets`:
 ```python
 import cabinets
 
-new_obj = cabinets.read('file://test.json')
+new_obj = cabinets.read('test.json')
 ```
 
 That's it! The file is *loaded* and *parsed* in just one line.
 
 ### Write a file
 
-`cabinets` also supports creating files. We can rewrite the first example using only `cabinets`.
+`cabinets` also supports creating files. We can rewrite the first example using
+only `cabinets`.
 
 ```python
 import cabinets
 
 obj = {'test': 1}
-cabinets.create('file://test.json', obj)
+cabinets.create('test.json', obj)
 
-new_obj = cabinets.read('file://test.json')
+new_obj = cabinets.read('test.json')
 
 assert new_obj == obj
 ```
 
+### Reading and Writing with Other Protocols
+
+In the previous examples we saw `read` and `write` operations on local files using the
+default *file* protocol. However, one of the main benefits of using `cabinets` is the
+flexibility to read and write files using multiple different protocols, with a common
+interface.
```

Comment:
```suggestion
Using `cabinets` allows you to interact with multiple file storage protocols
depending on the URI you specify. In the previous examples, we used
`read()` and `write()` to operate within our local file system; that's
because `cabinets` assumes we're using the `file://` protocol by default.
Luckily, accessing other storage systems is just as easy!
```